backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/hu/M_HU_StepDef.java

```diff
@@ -237,10 +238,13 @@ public void find_HUs(final int timeoutSec, @NonNull final DataTable dataTable) t
 		for (final Map<String, String> tableRow : dataTable.asMaps())
 		{
 			final String inventoryLineIdentifier = DataTableUtil.extractStringForColumnName(tableRow, I_M_InventoryLine.COLUMNNAME_M_InventoryLine_ID + "." + TABLECOLUMN_IDENTIFIER);
+			final Integer inventoryLineId = inventoryLineTable.getOptional(inventoryLineIdentifier)
+					.map(I_M_InventoryLine::getM_InventoryLine_ID)
+					.orElseGet(() -> Integer.parseInt(inventoryLineIdentifier));
```

Comment:
from a quick search for `s, there are added M_HUs for inventory` among feature files, we can AFAIU drop
the fallback to `Integer.parseInt(inventoryLineIdentifier));`. everyone just uses the actual identifier